backend/azure/dtmb/internal/backend/v1/client.proto

```diff
@@ -0,0 +1,115 @@
+syntax = "proto3";
```

Comment:
I assume these protos are here temporarily right? Eventually, they'll be in their own repos?

Reply:
Yes - I am waiting to hear from you where we'll have the protos :)